configs/VRBF405/config.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+
+#define BOARD_NAME        VRBF45
+#define MANUFACTURER_ID   VRBA
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACCGYRO_BMI270
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+
+#define BEEPER_PIN PB2
+#define MOTOR1_PIN PB0
+#define MOTOR2_PIN PB1
+#define MOTOR3_PIN PB5
+#define MOTOR4_PIN PB4
+#define MOTOR5_PIN PB6
+#define MOTOR6_PIN PB7
+#define MOTOR7_PIN PC8
+#define MOTOR8_PIN PC9
+#define RX_PPM_PIN PA3
+#define LED_STRIP_PIN PA8
+#define UART1_TX_PIN PA9
+#define UART2_TX_PIN PA2
+#define UART3_TX_PIN PB10
+#define UART4_TX_PIN PA0
+#define UART6_TX_PIN PC6
+#define UART1_RX_PIN PA10
+#define UART2_RX_PIN PA3
+#define UART3_RX_PIN PB11
+#define UART4_RX_PIN PA1
+#define UART5_RX_PIN PD2
+#define UART6_RX_PIN PC7
+#define I2C1_SCL_PIN PB8
+#define I2C1_SDA_PIN PB9
+#define LED0_PIN PC14
+#define SPI1_SCK_PIN PA5
+#define SPI2_SCK_PIN PB13
+#define SPI3_SCK_PIN PC10
+#define SPI1_SDI_PIN PA6
+#define SPI2_SDI_PIN PB14
+#define SPI3_SDI_PIN PC11
+#define SPI1_SDO_PIN PA7
+#define SPI2_SDO_PIN PB15
+#define SPI3_SDO_PIN PC12
+#define ESCSERIAL_PIN PD2
+#define CAMERA_CONTROL_PIN PB3
+#define ADC_VBAT_PIN PC2
+#define ADC_CURR_PIN PC1
+#define PINIO1_PIN PC13
+#define FLASH_CS_PIN PA15
+#define MAX7456_SPI_CS_PIN PB12
+#define GYRO_1_EXTI_PIN PC4
+#define GYRO_1_CS_PIN PA4
+#define USB_DETECT_PIN PC5
+
+#define TIMER_PIN_MAPPING \
```

Comment:
Motor 4 DMA assignment conflicts with SPI2_TX, which means OSD will perform poorly.
Motors 1 and 3 DMA conflict with SPI3_TX, which means flash will perform poorly. 

These motor outputs should be changed to avoid SPI DMA conflicts.